src/mono/netcore/System.Private.CoreLib/src/System/Threading/WaitSubsystem.HandleManager.Unix.cs

```diff
@@ -0,0 +1,56 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
```

Comment:
Have you done any changes to these files? For review, I would be useful to have separate commit (or even PR) that just copies the files over, so that it easy to review any changes you have done on top of it.

Reply:
I have indeed made changes, mostly the annotations but also removing some CoreRT-specific invocations. Happy to go back and split the commit up, it was just annoying to do initially since I was fixing a host of build errors.

Reply:
The most up to date version of these files is in https://github.com/dotnet/runtimelab/tree/feature/NativeAOT/src/coreclr/nativeaot/System.Private.CoreLib/src/System/Threading. Not sure whether you have copied them from there, or somewhere else.

Reply:
>  I would be useful to have separate commit (or even PR)

Looks like we may be iterating on this a bit, so I would recommend to submit PR that just copies the needed files over, without any changes and without hooking it to the build. And then separate PR to hook it up to the build and make the necessary adjustments.